### PR TITLE
Fix telemetry

### PR DESCRIPTION
**Pull request by kendru**
The telemetry changes caused exo to silently fail, so the merge was reverted. This PR contains the telemetry code and will soon have the fix applied.

### CODE REVIEW

**Comment by kendru** [comment]
@brandonbloom I ran with this branch yesterday without any issues. Could you do a sanity check again to see if it is still busted for you?

**Comment by brandonbloom** [comment]
I just did:

```
git fetch --all
git checkout -b fix-telemetry origin/fix-telemetry
git merge origin/main
./script/install-release.sh
exo daemon
```

And got as output: `daemon not healthy`

**Comment by kendru** [comment]
@brandonbloom The problem was that the server was not starting fast enough because it was trying to start the session synchronously. I just moved that into a go routine, since any call to send an event will not be scheduled until the session is actually initialized.

File: internal/telemetry/default.go [hunk 1]
```diff
@@ -36,7 +36,9 @@ func (t *defaultTelemetry) IsEnabled() bool {
 }
 
 func (t *defaultTelemetry) StartSession(ctx context.Context) {
-	t.ensureSession(ctx)
+	go func() {
+		t.ensureSession(ctx)
+	}()
```
Comment (brandonbloom) [inline internal/telemetry/default.go:41]: this is just `go t.ensureSession(ctx)`

**Pull request merged**